client/types/disputes.d.ts

```diff
@@ -11,8 +11,21 @@ interface Evidence {
 }
 
 interface EvidenceDetails {
+	/**
+	 * Whether evidence has been staged for this dispute.
```

Comment:
I wasn't sure what `has_evidence` represented, so I've added explanatory comments to this TS interface to improve developer experience.